src/runtime/GlobalObjectBuiltinIntl.cpp

```diff
@@ -525,7 +522,7 @@ static Value builtinIntlPluralRulesResolvedOptions(ExecutionState& state, Value
         i++;
     } while (true);
 
-    options->defineOwnPropertyThrowsException(state, ObjectPropertyName(state, Value(String::fromASCII("pluralCategories"))), ObjectPropertyDescriptor(pluralCategories, ObjectPropertyDescriptor::AllPresent));
+    options->defineOwnPropertyThrowsException(state, ObjectPropertyName(AtomicString(state.context(), "pluralCategories")), ObjectPropertyDescriptor(pluralCategories, ObjectPropertyDescriptor::AllPresent));
```

Comment:
Why didn't you convert "pluralCategories" into lazy string?

Reply:
"pluralCategories" is only used here.
and resolvedOptions function is called rarely I think(this function is not used on initializion or formatting).
so I didn't want use one pointer space for "pluralCategories"

Reply:
I see. BTW how about using `String::FromExternalMemory` flag to avoid any string copy for this case?

Reply:
Good idea!

Reply:
I added it